test/rails_app/config/application.rb

```diff
@@ -0,0 +1,32 @@
+require File.expand_path('../boot', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.